lexxpluss_apps/src/rosserial_tof.hpp

```diff
@@ -50,7 +50,7 @@ class ros_tof {
     }
 private:
     std_msgs::Float64MultiArray msg;
-    float msg_data[2];
+    double msg_data[2];
```

Comment:
The type had been changed. Need to respond to that.
It is appropriate to change from float to double for Float64MultiArray.h.

```diff
namespace std_msgs
{

  class Float64MultiArray : public ros::Msg
  {
    public:
      typedef std_msgs::MultiArrayLayout _layout_type;
      _layout_type layout;
      uint32_t data_length;
-      typedef float _data_type;
+      typedef double _data_type;
      _data_type st_data;
      _data_type * data;
```


https://github.com/LexxPluss/ros_msgs/commit/cf7716254c6d8a7fe8763533904860f2dca8fe2a#diff-9157bb2fecbb1d253e675b58cdfe7289bf267839e5c762af1c7338a1f4d57e18R19